<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VEEDSelectvar</class>
 <widget class="QWidget" name="VEEDSelectvar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="tv_left">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>171</width>
     <height>421</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tv_right">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>211</width>
     <height>451</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>500</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>500</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
